docs/getting_started.md

```diff
@@ -49,7 +50,7 @@ def run_pipeline( # (1)!
     )
 
     return {
-        "loss": validation_error, #! (2)
+        "loss": validation_error,  # ! (2)
```

Comment:
Did you check that adding this leading whitespace still leads to correct rendering with the docs? You'd have to run `mkdocs serve` and check.

Reply:
Thanks for letting me know! I checked the generated docs and compared with the current ones and I see no issues with the rendered page after introduction of this whitespace. I think this was introduced during `pre-commit run`